tests/webview/test_google_redirect.py

```diff
@@ -0,0 +1,18 @@
+from requests import get
+from tests import markers
+
+
+@markers.webview
+@markers.nondestructive
+def test_google_redirect_archive_results(webview_base_url, chrome_options):
+    # end-to-end integration test for redirecting google away from archive URLs
+    # see: https://github.com/openstax/cnx/issues/209
+
+    url = "https://archive-qa.cnx.org/contents/fea3130c-6e57-41b2-a00f-0267ffae273c@5"
```

Comment:
You should be able to build this with the proper fixture. `archive_base_url` should work.

Reply:
You'll also want to parametrize the uuid and add an identifier.  A documentation reference is this: https://docs.pytest.org/en/latest/parametrize.html

Code examples within the framework are here:
https://github.com/openstax/cnx-automation/blob/master/tests/webview/ui/test_content.py#L218

The url can be provided by the `archive_base_url` fixture and example of that is here:
https://github.com/openstax/cnx-automation/blob/511e151d9e19adcb4288f05ce973e00b0372cd5c/tests/webview/ui/test_robots.py#L22

Reply:
Not exactly why, in this case, I need to parametrize but—done. 💯